### DISCUSSION TITLE
No GUI for energy saving in SpiralLinux LXQt

### DISCUSSION THREAD

**Discussion by help-2024** [SpiralLinux/SpiralLinux-project#431] [Q&A]
Hello. 
The monitor goes off automatically after 10 minutes of inactivity and the PC goes into hibernation after 1 hour. Unfortunately this setting cannot be changed. 
 
You can try to enable and configure the "idleness watcher" in the energy management of LXQt settings, but SpiralLinux ignores any changes... 
Please someone suggests me a **GUI to install**? 
 
Thanks

**Comment by geckolinux** [comment]
> the PC goes into hibernation after 1 hour

That's not the default setting that comes with SpiralLinux.

Changes to the default settings shown here have been tested to work via the GUI, including power settings:

![image](https://github.com/SpiralLinux/SpiralLinux-project/assets/2692138/7768c17b-a89c-45df-8b3e-76918a90f235)

The only thing not covered in the LXQt GUI settings is the time that it takes to automatically lock the screen, which you can disable by unchecking the highlighted Autostart item in the screenshot above.

**Comment by help-2024** [comment]
Thank You for the precious screenshot. 
I'm not sure, it is perhaps a suspension, not a hibernation. But I'm sure it happens after 1 hour! 
 
Anyway what I described happens with the following settings: 
- on "power management / idle / idleness watcher" options AC and Battery are both set to "nothing"!! 
- on "session settings / autostart" the settings are identical to your screenshot. 
 
So it seems that LXQt GUI does not work for some mysterious reason... 
:-(

**Reply by geckolinux** [reply]
You might need to log out and log in again after changing the **Idleness Watcher** settings. But they do work.

> on "session settings / autostart" the settings are identical to your screenshot

You should disable the **Screen Locker** autostart item if you don't want it to lock the screen after 1 hour.

**Comment by help-2024** [comment]
Okay, half a problem has disappeared. 
I disabled the Screen Locker in "session settings / autostart", then I rebooted the computer and it no longer goes into hibernation after 1 hour of inactivity. 
Thank You so much for having revealed the meaning of this option. 
 
As for the monitor that turns off after 10 minutes, unfortunately everything is like before. 
I tried to set "power management / idle / idleness watcher" at 30 minutes with **Turn Off Monitor(s)** and rebooted the PC, with **Lock Screen** and rebooted the PC, with **Nothing** and rebooted the PC, finally I completely **disabled** the "idleness watcher" and rebooted the PC... 
In all cases just mentioned, the monitor goes off automatically after 10 minutes of inactivity. 
 
Therefore it seems my LXQt GUI does not work for some mysterious reason.

**Reply by geckolinux** [reply]
Thanks for your report. First of all I think we need to clarify some terms:
- **suspend** = freeze the entire computer in a low-power sleep state, active operating system RAM contents are maintained
- **hibernate** = completely power off the entire computer, RAM contents from last operating system session stored on the hard drive
- **screen off** = just the screen (monitor) goes black and turns off via a system known as Display Power Management Signaling (DPMS), but the operating system remains active
- **lock screen** = computer is fully on, but forces the currently active user to enter the password again to continue using the system

So disabling the **Screen Locker** autostart item disables the automatic **lock screen** after 1 hour.

I understand that you also want to disable the automatic **screen off** function after 10 minutes. It looks like there is a conflict between several different systems that prevents the LXQt GUI from changing / disabling DPMS. This was later fixed for LXQt:
https://github.com/lxqt/lxqt/issues/494
https://github.com/lxqt/lxqt-powermanagement/pull/375
However, Debian Stable uses an older version of `lxqt-powermanagement` from before the fix was released.

So, to force it to **disable** the automatic **screen off / DPMS** system please open the LXQt **Session Settings** GUI, and in the **Autostart** section do this:
1. **[Add]**
2. **Name:** Disable display power management
3. **Command:** `xset s off dpms 0 0 0`
4. **[OK]**
5. Log out and log in again

**Comment by help-2024** [comment]
Perfect. 
Now everything is disabled. 
 
Finally the whole situation is clear. Thank You!
